pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[tool.poetry]
+name = "django-multi_import"
+version = "2.0.0"
+homepage = "https://github.com/sdelements/django-multi-import"
+description = "Import/export multi Django resources together atomically."
+authors = ["Security Compass <contact@securitycompass.com>"]
+license = "MIT"
+readme = "README.md"
+# See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers=[
+    # How mature is this project? Common values are
+    #   3 - Alpha
+    #   4 - Beta
+    #   5 - Production/Stable
+    'Development Status :: 5 - Production/Stable',
+
+    # Indicate who your project is intended for
+    'Intended Audience :: Developers',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+
+    # Pick your license as you wish (should match "license" above)
+    'License :: OSI Approved :: MIT',
+
+    # Supported Languages
+    'Natural Language :: English',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.12',
+    'Framework :: Django',
+]
+packages = [
+  { include = "multi_import" },
+  { include = "tests", format = "sdist" },
+]
+exclude = [
+  "multi_import/**/tests",
+  "tests"
+]
+
+[tool.poetry.dependencies]
+python = "~3.12"
+django = "~4.2"
+djangorestframework = "3.15.1"
+chardet = "5.2.0"
+tablib = {extras = ["all"], version = "3.6.1"}
+pyyaml = "6.0.1"
```

Comment:
Could we loosen these constraints to the nearest minor version? Will make minor bumps of these dependencies easier since we won't have to go and bump the constraints in this package

Reply:
Updated - https://github.com/sdelements/django-multi-import/pull/40/commits/2918fbece26487cd4768c6148700d24d468bdeb7